huaweicloud/services/acceptance/fgs/resource_huaweicloud_fgs_function_test.go

```diff
@@ -184,6 +184,43 @@ func TestAccFgsV2Function_agency(t *testing.T) {
 	})
 }
 
+func TestAccFgsV2Function_fgsVersion(t *testing.T) {
+	var f function.Function
+	randName := acceptance.RandomAccResourceName()
+	resourceName := "huaweicloud_fgs_function.test"
+
+	dc := acceptance.InitResourceCheck(
+		resourceName,
+		&f,
+		getResourceObj,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.TestAccPreCheck(t) },
+		Providers:    acceptance.TestAccProviders,
+		CheckDestroy: dc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFgsV2Function_fgsVersion(randName),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "functiongraph_version", "v2"),
```

Comment:
suggest to add this verstion check for `v1` to basic test as well, then we can notice that when the default version changed on service side.

Reply:
done.